be/src/common/status.h

```diff
@@ -22,7 +22,7 @@ namespace doris {
 
 // ErrorName, ErrorCode, String Description, Should print stacktrace
 #define APPLY_FOR_ERROR_CODES(M) \
-    M(OLAP_SUCCESS, 0, "", true) \
+    M(OLAP_SUCCESS, 0, "", false) \
```

Comment:
Why change this?

Reply:
Because we don't need to print stack trace for `OLAP_SUCCESS`